src/go/pt-pg-summary/main.go

```diff
@@ -17,6 +17,10 @@ import (
 	"github.com/percona/percona-toolkit/src/go/pt-pg-summary/templates"
 )
 
+const (
+	TOOLNAME = "pt-pg-summary"
```

Comment:
Go convention doesn't use all caps names for constants: https://github.com/golang/go/wiki/CodeReviewComments#mixed-caps

It should be `Toolname` if it's public const, or most likely just `toolname`

Reply:
Done